beancount/reports/price_reports.py

```diff
@@ -167,8 +167,8 @@ class TickerReport(base.TableReport):
     names = ['tickers', 'symbols']
 
     def generate_table(self, entries, errors, options_map):
-        commodity_map = getters.get_commodity_map(entries)
-        ticker_info = getters.get_values_meta(commodity_map, 'name', 'ticker', 'quote')
+        commodities = getters.get_commodity_directive(entries)
```

Comment:
Hmm... typo?

Reply:
Yes, I wonder how I managed to fat finger this between the moment in which I run the tests and the one in which I committed the changes... Fixup on its way

Reply:
Ah! No tests cover this code, that's why.

Reply:
Current tests don't pass either.

Reply:
self = <beancount.plugins.check_commodity_test.TestCheckCommodity testMethod=test_get_commodity_map_ex>

Reply:
Ops. Should be fixed.